google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/NavigateSearchResultPagesCachingTokens.java

```diff
@@ -140,25 +139,25 @@ private static void runExample(GoogleAdsClient googleAdsClient, long customerId)
       System.out.printf("Total number of campaigns found: %d.%n", totalNumberOfResults);
 
       // Determines the total number of pages and prints it.
-      totalNumberOfPages = (int) Math.ceil((double) totalNumberOfResults / (double) PAGE_SIZE);
+      totalNumberOfPages = (int) Math.ceil(totalNumberOfResults / (double) PAGE_SIZE);
       System.out.printf("Total number of pages: %d.%n", totalNumberOfPages);
       if (totalNumberOfPages == 0) {
         System.out.println("Could not find any campaigns.");
-        System.exit(1);
+        return;
       }
     }
 
     // Demonstrates how the logic works when iterating pages forward. We select a page that is
```

Comment:
One last "we" sentence. How about:

`Demonstrates how the logic works when iterating pages forward. Selects a page...`

Reply:
There are actually a bunch of these throughout the example. I know I changed the language in the last example, but is this something we feel strongly about? I do think we've used the "we" phrasing in the past, and it may become a little burdensome updating all the comments from golden CLs moving forward if Java is the only library abiding by this.